luigi_pipeline/lib/hail_tasks.py

```diff
@@ -72,6 +81,63 @@ def sample_type_stats(mt, genome_version, threshold=0.3):
             ht_stats['match'] = (ht_stats['matched_count']/ht_stats['total_count']) >= threshold
         return stats
 
+    @staticmethod
+    def subset_samples(mt, subset_path):
+        """
+        Subset the MatrixTable to the provided list of samples
+        :param mt: MatrixTable from VCF
+        :param subset_path: Path to a file with a single column 's'
+        :return: MatrixTable subsetted to list of samples
+        """
+        subset_ht = hl.import_table(subset_path, no_header=False).key_by('s')
```

Comment:
`import_table` accepts a `key` parameter. https://hail.is/docs/0.2/methods/impex.html#hail.methods.import_table

Same comment for `remap_sample_ids`.

Reply:
Didn't know that, thanks!